F21/rishith-cmykt/README

```diff
@@ -1 +1,23 @@
 Your program for the code review lab goes in this directory
+#ENGLSIH LANGUAGE
+y=input("Enter any word in uppercase letters : ")
+c=y.lower()
+print("Lower case letters : ",c)
+#IRISH LANGUAGE
+r = input("Enter any word in irish capital letters : ")
+if (r[0] == 'n' or r[0] == 't') and (r[1] == 'A' or r[1] == 'E' or r[1] == 'I' or r[1] == 'O' or r[1] == 'U' or r[1] == 'Á' or r[1] == 'É' or r[1] =='Í' or r[1] =='Ó' or r[1] == 'Ú'):
```

Comment:
If the irish word the user inputs was only one character long this would throw an error when trying to index 1

Reply:
Although the check for the second letter to be one of those uppercase letter, the line is too long. I would consider creating a list or string of those characters and say something like r[1] in 'AEIO".